CHANGELOG.md

```diff
@@ -17,11 +17,15 @@ and this project adheres to
 ## Changed
 
 - 🔨(frontend) encapsulated title to its own component #474
-- 🐛(frontend) Fix hidden menu on Firefox #468
 - ⚡️(backend) optimize number of queries on document list view #411
 - ♻️(frontend) stop to use provider with version #480
 - 🚚(collaboration) change the websocket key name #480
 
+## Fixed
+
+- 🐛(frontend) Fix hidden menu on Firefox #468
```

Comment:
this line is wrongly committed

Reply:
I think its ok, I don't want to make a commit for that.